assets/js/base/utils/price.js

```diff
@@ -5,25 +5,21 @@ import { sprintf } from '@wordpress/i18n';
 import { CURRENCY } from '@woocommerce/settings';
 
 /**
- * Format a price with currency data.
+ * Format a price, provided using the smallest unit of the currency, as a
+ * decimal complete with currency symbols using current store settings.
  *
- * @param {number} value Number to format.
- * @param {string} priceFormat  Price format string.
- * @param {string} currencySymbol Currency symbol.
+ * @param {number} price Price, in cents, to format.
  */
-export const formatPrice = (
-	value,
-	priceFormat = CURRENCY.priceFormat,
-	currencySymbol = CURRENCY.symbol
-) => {
```

Comment:
Why this change?  Originally this format helper defaulted to set currency values from the server but still allowed for overriding for specific formats.  Now this ONLY uses currency values from the server.  This change also breaks tests for the utility function.

Reply:
Pretty sure the tests were only added after I changed this, before I rebased to master.

I changed it because the only usage of this util needed server defined values, and since API endpoints handling money all use server values I didn't see much benefit to allowing overrides.

Reply:
I'll let you make the call.  It's not a publicly exposed api, so either way it should be fine.  Ya I think the tests were added in a pull Albert did fairly recently. Not much involved in updating them :).